tests/old_tests/issue3/testi.py

```diff
@@ -1,14 +1,21 @@
 # -*- coding: utf-8; -*-
 # See issue #3
 
+
 def f():
     return [x for x in range(10)]
 
+
 def g():
     return [(x, y) for x in range(10) for y in range(10)]
 
-def h():
-    return [([(name, allargs) for name, _, _, allargs, _ in recs],
-              {name: inargs for name, inargs, _, _, _ in recs},
-              {name: meta for name, _, _, _, meta in recs})
-            for recs in (results[key] for key in sorted(results.keys()))]
+
```

Comment:
Why is the definition of `h` commented out?

(This code isn't meant to run - it's just a snippet to feed into Pyan's analyzer, see issue #3. FWIW, that code comes from [here](https://github.com/TUTElectromechanics/mm-codegen/blob/master/fian.py#L256), which I did a couple of years ago for a simple optimizing [SymPy to Fortran compiler](https://github.com/TUTElectromechanics/mm-codegen) to generate [ElmerFEM](https://github.com/ElmerCSC/elmerfem) plugins.)

Reply:
The issue is that `results` is not defined. I added it to the function definition to ensure it passes the linters.

Reply:
Ok. Thanks!